torch/_higher_order_ops/cond.py

```diff
@@ -449,8 +450,8 @@ def cond_func(ctx, pred, true_fn, false_fn, inputs):
     unwrapped_inputs = ctx.unwrap_tensors(inputs)
     unwrapped_pred = ctx.unwrap_tensors(pred)
     with ctx.redispatch_to_next() as m:
-        functional_true = ctx.functionalize(true_fn)
-        functional_false = ctx.functionalize(false_fn)
+        functional_true = ctx.functionalize(_maybe_run_with_interpreter(true_fn))
```

Comment:
We could have a ctx.functionalize_wrap_with_interpreter or something but i guess it's OK to do it manually for hops since people usually just copy/paste over existing code lol.